### ISSUE TITLE
Images not Loading

### ISSUE THREAD

**Issue by Paulkendrick2126** [Paulkendrick2126/Redesigned-Uni-Website#1]
The images in the "Our Global Campus Category" is not loading via the hosted Github Link(https://paulkendrick2126.github.io/Redesigned-Uni-Website/), but when I view directly from my IDE via live server in my local browser it opens just fine.

**Comment by zablon-oigo** [comment]
Hello, 
I would love to contribute to this issue

**Comment by Paulkendrick2126** [comment]
> Hello, I would love to contribute to this issue

Hi Zablon, thanks for responding. I'm new to Github in fact I'm a beginner programmer so pardon me.

So how do we go about it, first off can I ask if you understand the issue and have you checked the code and viewed it?

**Comment by zablon-oigo** [comment]
Yes, I have checked your code, I assume you're trying to build something similar to a certain college site, please correct me if I am wrong.
your **style.css file , background-image:url()** seem to be missing opening and closing mark 
**background-image: url('banner.jpg')**
If not please expound more on how the image is not displaying, where in particular are you trying to render the image, kindly

**Comment by Paulkendrick2126** [comment]
Yeah, you're right. Just a Static Website. Oh my, thanks a lot. I've corrected it, thank you. 

OK, I'd like you to open the link(https://paulkendrick2126.github.io/Redesigned-Uni-Website/). I have attached media files for you to understand better. The images in the "our global campus." section fail to render. I used the console dev tools to see what went wrong and realized the files ( London, Washington and New York) somehow are not rendered by the browser, but other images are. 
![IMG-20231103-WA0030](https://github.com/Paulkendrick2126/Redesigned-Uni-Website/assets/118655112/90a24c28-0ad4-401d-86a1-2a7aabb2aca8)
![IMG-20231103-WA0032](https://github.com/Paulkendrick2126/Redesigned-Uni-Website/assets/118655112/9a4eb2b1-8613-42ac-9ac7-c89c8e6973d9)
![IMG-20231103-WA0034](https://github.com/Paulkendrick2126/Redesigned-Uni-Website/assets/118655112/6e4675e0-80a5-4286-8830-67dcf61c61c8)
![IMG-20231103-WA0021](https://github.com/Paulkendrick2126/Redesigned-Uni-Website/assets/118655112/2ce1bc98-1ef7-467e-b900-f8b22594f817)


Please let me know should you want any clarity, information, file or anything. Thank you.

**Comment by Paulkendrick2126** [comment]
Hi, I was able to solve it, thanks to SO. I removed the '/' at the beginning of the src  .  Apparently, GitHub is pretty sensitive.

Thanks a lot for reaching out and working with me. Do you have any social media for me to follow you on ?

**Comment by zablon-oigo** [comment]
Glad to hear that you were able to solve it, I checked the link you shared
and I am impressed the site looks amazing with all the animation and it's
responsive to smaller screens.

*Good Work Paul*You can always get in touch in case you face another issue
or if you need clarity on certain topics.
You can follow me on GitHub check some of my latest work or LinkedIn at
https://www.linkedin.com/in/zablon-oigo-ba4268234/
Have a great day
